core/src/main/java/org/infinispan/commands/functional/ReadWriteManyCommand.java

```diff
@@ -180,11 +181,21 @@ public boolean alwaysReadsExistingValues() {
    }
 
    @Override
+   public long getFlagsBitSet() {
+      return EnumUtil.EMPTY_BIT_SET;
+   }
+
+   @Override
    public void setFlags(Set<Flag> flags) {
       // TODO: Customise this generated block
```

Comment:
No idea why the TODOs are there, but there are a bunch of methods like this. Not sure if the class is completed or not.